### PR TITLE
Run e2e tests in FRA1

### PR DESCRIPTION
**Pull request by mboukhalfa**
Run e2e PR tests in Frankfurt region

### CODE REVIEW

File: jenkins/scripts/integration_test.sh [hunk 1]
```diff
@@ -50,9 +50,9 @@ VM_TIMELABEL="${VM_TIMELABEL:-$(date '+%Y%m%d%H%M%S')}"
 TEST_EXECUTER_VM_NAME="${TEST_EXECUTER_VM_NAME:-ci-test-vm-${VM_TIMELABEL}}"
 TEST_EXECUTER_PORT_NAME="${TEST_EXECUTER_PORT_NAME:-${TEST_EXECUTER_VM_NAME}-int-port}"
 
-# Run feature tests and main, master and release* tests in the Frankfurt region
-if [[ "${TESTS_FOR}" == "feature_tests"* ]] || [[ "${UPDATED_BRANCH}" == "release"* ]] || \
-   [[ "${UPDATED_BRANCH}" == "main" ]] || [[ "${UPDATED_BRANCH}" == "master" ]] 
+# Run feature tests, e2e tests, main, master and release* tests in the Frankfurt region
+if [[ "${TESTS_FOR}" == "feature_tests"* ]] || [[ "${UPDATED_BRANCH}" == "release"* ]] || [[ "${TESTS_FOR}" == "e2e_tests"* ]] || \
+   [[ "${UPDATED_BRANCH}" == "main" ]] || [[ "${UPDATED_BRANCH}" == "master" ]] || [[ "${UPDATED_BRANCH}" == "release"* ]]
```
Comment (fmuyassarov) [inline jenkins/scripts/integration_test.sh:55]:
```suggestion
   [[ "${UPDATED_BRANCH}" == "main" ]] || [[ "${UPDATED_BRANCH}" == "master" ]]
```
it is there already

File: jenkins/scripts/integration_delete.sh [hunk 2]
```diff
@@ -19,9 +19,9 @@ source "${CI_DIR}/utils.sh"
 
 TEST_EXECUTER_PORT_NAME="${TEST_EXECUTER_PORT_NAME:-${TEST_EXECUTER_VM_NAME}-int-port}"
 
-# Run feature tests and main, master and release* integration tests in the Frankfurt region
-if [[ "${TESTS_FOR}" == "feature_tests"* ]] || [[ "${UPDATED_BRANCH}" == "release"* ]] || \
-   [[ "${UPDATED_BRANCH}" == "main" ]] || [[ "${UPDATED_BRANCH}" == "master" ]]
+# Run feature tests, e2e tests, main, master and release* tests in the Frankfurt region
+if [[ "${TESTS_FOR}" == "feature_tests"* ]] || [[ "${UPDATED_BRANCH}" == "release"* ]] || [[ "${TESTS_FOR}" == "e2e_tests"* ]] || \
+   [[ "${UPDATED_BRANCH}" == "main" ]] || [[ "${UPDATED_BRANCH}" == "master" ]] || [[ "${UPDATED_BRANCH}" == "release"* ]]
```
Comment (fmuyassarov) [inline jenkins/scripts/integration_delete.sh:24]: same

File: jenkins/scripts/integration_clean.sh [hunk 3]
```diff
@@ -49,9 +49,9 @@ function cleanup() {
   echo "Old leftover resources are cleaned successfully!"
 }
 
-# Run feature tests and main, master and release* integration tests in the Frankfurt region
-if [[ "${TESTS_FOR}" == "feature_tests"* ]] || [[ "${UPDATED_BRANCH}" == "release"* ]] || \
-   [[ "${UPDATED_BRANCH}" == "main" ]] || [[ "${UPDATED_BRANCH}" == "master" ]]
+# Run feature tests, e2e tests, main, master and release* tests in the Frankfurt region
+if [[ "${TESTS_FOR}" == "feature_tests"* ]] || [[ "${UPDATED_BRANCH}" == "release"* ]] || [[ "${TESTS_FOR}" == "e2e_tests"* ]] || \
+   [[ "${UPDATED_BRANCH}" == "main" ]] || [[ "${UPDATED_BRANCH}" == "master" ]] || [[ "${UPDATED_BRANCH}" == "release"* ]]
```
Comment (fmuyassarov) [inline jenkins/scripts/integration_clean.sh:54]: same

File: jenkins/scripts/fetch_logs.sh [hunk 4]
```diff
@@ -20,9 +20,9 @@ source "${CI_DIR}/utils.sh"
 
 TEST_EXECUTER_PORT_NAME="${TEST_EXECUTER_PORT_NAME:-${TEST_EXECUTER_VM_NAME}-int-port}"
 
-# Run feature tests and main, master and release* integration tests in the Frankfurt region
-if [[ "${TESTS_FOR}" == "feature_tests"* ]] || [[ "${UPDATED_BRANCH}" == "release"* ]] || \
-   [[ "${UPDATED_BRANCH}" == "main" ]] || [[ "${UPDATED_BRANCH}" == "master" ]]
+# Run feature tests, e2e tests, main, master and release* tests in the Frankfurt region
+if [[ "${TESTS_FOR}" == "feature_tests"* ]] || [[ "${UPDATED_BRANCH}" == "release"* ]] || [[ "${TESTS_FOR}" == "e2e_tests"* ]] || \
+   [[ "${UPDATED_BRANCH}" == "main" ]] || [[ "${UPDATED_BRANCH}" == "master" ]] || [[ "${UPDATED_BRANCH}" == "release"* ]]
```
Comment (fmuyassarov) [inline jenkins/scripts/fetch_logs.sh:25]: same

**Review by fmuyassarov — CHANGES_REQUESTED** [review]

File: jenkins/scripts/integration_clean.sh (hunk 3 above)
Comment (smoshiur1237) [reply jenkins/scripts/integration_clean.sh:54]: line 52-54 is not necessary. I have changes them in #273, I have deleted them and add cleanup separately for FRA1

Comment (smoshiur1237) [reply jenkins/scripts/integration_clean.sh:54]: @mboukhalfa ^^

Comment (smoshiur1237) [reply jenkins/scripts/integration_clean.sh:54]: Please dont add anyting to this file. Because it is running once in a day for cleanup

**Review by smoshiur1237 — CHANGES_REQUESTED** [review]

**Review by namnx228 — COMMENTED** [review]
LGTM

**Comment by smoshiur1237** [comment]
LGTM

**Comment by Rozzii** [comment]
LGTM

**Review by Rozzii — APPROVED** [review]

**Pull request merged**